cmd/destroy/remote.go

```diff
@@ -0,0 +1,213 @@
+package destroy
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	remoteBuild "github.com/okteto/okteto/cmd/build/remote"
+	oktetoErrors "github.com/okteto/okteto/pkg/errors"
+
+	"github.com/okteto/okteto/pkg/cmd/build"
+	"github.com/okteto/okteto/pkg/constants"
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/okteto/okteto/pkg/types"
+	"github.com/spf13/afero"
+)
+
+const (
+	dockerfileTemplate = `
+FROM {{ .OktetoCLIImage }} as okteto-cli
+
+FROM alpine as certs
+RUN apk update && apk add ca-certificates
+
+FROM {{ .UserDestroyImage }} as deploy
+
+ENV PATH="${PATH}:/okteto/bin"
+COPY --from=certs /etc/ssl/certs /etc/ssl/certs
+COPY --from=okteto-cli /usr/local/bin/* /okteto/bin/
+
+{{range $key, $val := .OktetoBuildEnvVars }}
+ENV {{$key}} {{$val}}
+{{end}}
+ENV {{ .NamespaceEnvVar }} {{ .NamespaceValue }}
+ENV {{ .ContextEnvVar }} {{ .ContextValue }}
+ENV {{ .TokenEnvVar }} {{ .TokenValue }}
+ENV {{ .RemoteDeployEnvVar }} true
+
+COPY . /okteto/src
+WORKDIR /okteto/src
+
+ENV OKTETO_INVALIDATE_CACHE {{ .RandomInt }}
+RUN okteto destroy {{ .DestroyFlags }}
+`
+	dockerignoreName = "deploy.dockerignore"
```

Comment:
Can we update to use the same `dockerignore` as deploy?

Reply:
yes, it has been uptaded after merge https://github.com/okteto/okteto/pull/3393 into this branch